lib/bundlex/toolchain/common/unix/os_deps.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Bundlex.Toolchain.Common.Unix.OSDeps do
+  @moduledoc false
+
+  require Logger
+  alias Bundlex.Output
+
+  @precompiled_path "#{Mix.Project.build_path()}/bundlex_precompiled/"
+
+  @spec get_flags(Bundlex.Native.t(), atom()) :: String.t()
+  def get_flags(native, flags_type) do
+    native.os_deps
+    |> Enum.flat_map(fn
+      {:pkg_config, lib_names} ->
+        get_flags_for_pkg_config(lib_names, flags_type, native.app)
+
+      {precompiled_dependency_path, lib_names} ->
+        get_flags_for_precompiled(
+          {precompiled_dependency_path, lib_names},
+          flags_type
+        )
+    end)
+    |> Enum.uniq()
+    |> Enum.join(" ")
+  end
+
+  defp parse_os_deps(os_deps) do
+    Enum.map(os_deps, fn
+      {precompiled_dependency, lib_name_or_names} ->
+        lib_names = Bunch.listify(lib_name_or_names)
+        {precompiled_dependency, lib_names}
+
+      lib_name ->
+        {:pkg_config, [lib_name]}
+    end)
+  end
+
+  defp get_flags_for_precompiled(
+         {{_precompiled_dependency_url, precompiled_dependency_path}, lib_names},
+         flags_type
```

Comment:
In this function second argument name is `flags_type`, while in `get_flags_for_pkg_config/3` it is `options`, but you pass in both places the same value

Reply:
done